packages/edit-navigation/src/components/editor/style.scss

```diff
@@ -1,39 +1,35 @@
 .edit-navigation-editor {
-	margin-top: $grid-unit-20;
+	margin: $grid-unit-20;
 
 	@include break-medium() {
 		align-items: flex-start;
 		display: flex;
 	}
 }
 
-.edit-navigation-editor__list-view,
-.edit-navigation-editor__visual-view {
-	background: $white;
-	border-radius: 3px;
-	border: $border-width solid $gray-200;
-}
-
 .edit-navigation-editor__list-view {
-	padding: $grid-unit-10;
+	padding-left: $grid-unit-20;
+	margin-left: $grid-unit-20;
+	border-left: 1px solid $gray-200;
 
 	@include break-medium() {
 		width: 300px;
 	}
 
+	h3 {
```

Comment:
Should probably give the `h3` a class name and use that instead.

Reply:
No strong preference.

Reply:
f4e9bb9bfb